### PR TITLE
Auth0 Customization Support

### PR DESCRIPTION
**Pull request by willronchetti**
- Small changes necessary to allow customization of Auth0 Domain and Allowed Connections
- NOTE: Addition of AUTH0_DOMAIN required in GAC

### CODE REVIEW

File: deploy/docker/production/cgap_any_alpha.ini [hunk 1]
```diff
@@ -1,8 +1,10 @@
 [app:app]
 use = config:base.ini#app
 session.secret = %(here)s/session-secret.b64
+auth0.domain = ${AUTH0_DOMAIN}
 auth0.client = ${AUTH0_CLIENT}
 auth0.secret = ${AUTH0_SECRET}
+auth0.allowed_connection = ${AUTH0_ALLOWED_CONNECTIONS}
```
Comment (netsettler) [inline deploy/docker/production/cgap_any_alpha.ini:7]: Should this be plural to match the env var?

**Review by netsettler — APPROVED** [review]
I'm not familiar with the protocol in detail, so if you want me to refer that we'd need to go over it interactively. But I didn't see anything obviously wrong here.

File: deploy/docker/production/cgap_any_alpha.ini (hunk 1 above)
Comment (willronchetti) [reply deploy/docker/production/cgap_any_alpha.ini:7]: Good catch this typo is actually a bug

**Comment by willronchetti** [comment]
This change just allows you to customize the Auth0Domain and the AllowedConnection portion of the configuration. The former allows you to work with Auth0 accounts that are not ours, and the latter allows you to use a custom set of connections configured with that Auth0 account that are not ours. I am not confident it will help the logout issue you're seeing, maybe we should discuss that further?

**Pull request merged**